lib/client/connection.js

```diff
@@ -584,3 +597,44 @@ Connection.prototype._firstQuery = function(fn) {
     }
   }
 };
+
+Connection.prototype._firstSnapshotRequest = function (fn) {
+  for (var id in this.snapshotRequests) {
+    var snapshotRequest = this.snapshotRequests[id];
+    if (fn(snapshotRequest)) {
+      return snapshotRequest;
+    }
+  }
+};
+
+/**
+ * Get a read-only snapshot at a given version or time
+ *
+ * @param collection - the collection name of the snapshot
+ * @param id - the ID of the snapshot
+ * @param version - the version number, or Date of the snapshot to fetch. If an exact version or Date match is not made,
+ *   then the next lowest version is returned. ie if a document has 6 versions, asking for v7 will return v6. If ops
```

Comment:
"then the next lowest version is returned" - should it be "then the next lower version is returned"?